packages/back-end/src/services/features.ts

```diff
@@ -101,21 +154,31 @@ export async function refreshSDKPayloadCache(
     const projectFeatures = key.project
       ? allFeatures.filter((f) => f.project === key.project)
       : allFeatures;
+    const projectExperiments = key.project
+      ? allVisualExperiments.filter((e) => e.experiment.project === key.project)
+      : allVisualExperiments;
 
-    if (!projectFeatures.length) continue;
+    if (!projectFeatures.length && !projectExperiments) continue;
```

Comment:
`projectExperiments` is an array, so it will always be truthy.  Need to do `.length` here too.

Reply:
Fixed